runtime-library/utils/type_utils.c

```diff
@@ -71,6 +73,12 @@ is_ts_closure_type(wasm_module_t wasm_module, wasm_defined_type_t type)
     field_type = wasm_struct_type_get_field_type(struct_type, 1, &mut);
     field_type_idx = field_type.heap_type;
     field_defined_type = wasm_get_defined_type(wasm_module, field_type_idx);
+    if (!wasm_defined_type_is_struct_type(field_defined_type)) {
+        return false;
+    }
+    field_type = wasm_struct_type_get_field_type(struct_type, 2, &mut);
```

Comment:
Can we define the `2` as a macro or enum?

Reply:
Yes, we can define a macro to represent the envParamLen in runtime, just as compiler.